src/surge-xt/gui/widgets/MenuCustomComponents.cpp

```diff
@@ -321,15 +347,17 @@ ModMenuCustomComponent::~ModMenuCustomComponent() noexcept = default;
 void ModMenuCustomComponent::getIdealSize(int &idealWidth, int &idealHeight)
 {
     auto menuText = source + " " + amount;
-    getLookAndFeel().getIdealPopupMenuItemSize(menuText, false, 20, idealWidth, idealHeight);
     auto h = idealHeight - 2 * mg;
+
+    getLookAndFeel().getIdealPopupMenuItemSize(menuText, false, 20, idealWidth, idealHeight);
```

Comment:
Why this change in behavior? I don't understand it.

Reply:
Reverted it!